includes/widgets/video.php

```diff
@@ -873,6 +873,7 @@ protected function register_controls() {
 			[
 				'label' => esc_html__( 'Content Width', 'elementor' ),
 				'type' => Controls_Manager::SLIDER,
+				'description' => esc_html__( 'This control was deprecated.', 'elementor' ),
```

Comment:
Please add an alert class to this description with alert type. In addition display this control for websites that already used this control only.

Reply:
Add copy team to change the text

Reply:
Done.